docs/sources/drill-down-metrics/index.md

```diff
@@ -0,0 +1,62 @@
+---
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Drill down your metrics
+weight: 30
+refs:
+  get-started:
+    - pattern: /docs/grafana/
+      destination: https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/metrics/get-started/
+    - pattern: /docs/grafana-cloud/
+      destination: https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/metrics/get-started/
+---
+
+# Drill down your metrics
+
+Drill down into your metrics to gain insight into your data without writing a query. First, filter metrics to appear in the **Metrics Drilldown** dashboard. Then, perform an in-depth analysis of selected metrics.
+
+## Filter the Metrics Drilldown dashboard
+
+To begin drilling down your data, filter the metrics that appear in the **Metrics Drilldown** dashboard.
+
+1. Navigate to the **Metrics Drilldown** page in Grafana. Refer to [Get started with Grafana Metrics Drilldown](ref:get-started).
+1. From the **Data source** dropdown, select a data source to view related metrics. Supported data sources include Prometheus and Prometheus-compatible data sources.
+
+     A dashboard of metrics for your selected data source appears.
+1. (Optional) Select a label name from the **Filter by label values** dropdown. Then, follow the prompts to complete your filter criteria.
+
+     {{< admonition type="note" >}}
+      You can apply multiple filters to your metrics.
+     {{< /admonition >}}
+1. (Optional) To search for metrics, type keywords in the search bar under **Search metrics**.
+1. Use the time picker to select a date and time range from the dropdown menu, or use an absolute time range.
+1. Click the down arrow next to the **Refresh** icon to set a refresh rate from the drop-down menu. The default refresh status is **Off**.
+
+The **Metrics Drilldown** dashboard adjusts to reflect your selected filters. To review the steps in your journey, use the **History** breadcrumbs at the top of the dashboard, as shown in the following screenshot.
```

Comment:
Has the history feature been removed for G12?

Reply:
The history component has been removed, this is correct. Now users can use their browser back button to navigate through their interactions. 

[PR for removing history here](https://github.com/grafana/metrics-drilldown/pull/221)